### ISSUE TITLE
Elektra-web selecting text is not possible on firefox

### ISSUE THREAD

**Issue by omnidan** [ElektraInitiative/libelektra#1851]
Unfortunately, I really cannot find a way to solve this in Firefox.

In webkit based browsers, I can [do `onDragStart={e => e.preventDefault()}` on a part of the element](https://github.com/react-dnd/react-dnd/issues/602#issuecomment-308267769) (in this case, only the text field) and it prevents dragging the element while still allowing text selection.

In Firefox, however, it seems to also prevent text selection, so there is no way to prevent dragging while still allowing text selection.

**Comment by markus2330** [comment]
Is it important to prevent dragging?

**Comment by omnidan** [comment]
@markus2330 if we don't prevent dragging, the whole key gets dragged instead of selecting text.

I want to be able to drag the whole key except when dragging in the text field, which is exactly what this solution does on webkit browsers, but not on firefox 😢

**Comment by markus2330** [comment]
Sorry, I cannot help.

Maybe @Namoshek or @vanessaNWK know something about it.

Otherwise asking the question somewhere else might help.

**Comment by markus2330** [comment]
Did you have luck by asking the question somewhere else?

**Comment by Namoshek** [comment]
Sorry, I forgot about this issue. Maybe disabling all default event handlers while a specific key (`ctrl` or `alt`) is pressed might work?

**Comment by omnidan** [comment]
@Namoshek I tried disabling drag & drop when `alt` is pressed. However, this requires a re-render of the whole tree view (not just single components of it), because all elements need to be set to not be draggable anymore. Unfortunately, this is a quite expensive operation which would get executed every time the alt key is pressed :/

My workaround here is that I added a button to copy the current value of a key. Using keyboard shortcuts to select and cut text also works. Selecting also works fine for me on latest Chrome:

![selecting](https://user-images.githubusercontent.com/668674/38130207-5e67601e-3402-11e8-832c-46ed26533930.gif)
